clayui.com/static/js/icons-autogenerated.json

```diff
@@ -679,6 +679,10 @@
         "name": "flag-full",
         "aliases": ["banner", "report"]
     },
+    {
+        "name": "du-be",
+        "aliases": [""]
+    },
```

Comment:
@pat270 maybe this should be in flags or is this intentional?

Reply:
Not intentional, let me figure out why it was placed there.

Reply:
@matuzalemsteles should be good now. There was a documentation error in the issue `du-BE` isn't a language code. It should be `nl-BE` which is already in Clay.

Reply:
Thanks @pat270